mlflow/langchain/utils/chat.py

```diff
@@ -39,12 +39,16 @@ def convert_lc_message_to_chat_message(lc_message: Union[BaseMessage]) -> ChatMe
     """
     if isinstance(lc_message, AIMessage):
         if tool_calls := _get_tool_calls_from_ai_message(lc_message):
+            content = lc_message.content
+            # For Anthropic model tool calls are returned twice so we need to filter them out from content
```

Comment:
Maybe, "A tool use content part is included in the content of Anthropic model tool calls"?